src/Popover/index.js

```diff
@@ -23,6 +23,11 @@ StyledActions.defaultProps = {
   theme: defaultTheme,
 };
 
+const StyledContentWrapper = styled.div`
+  overflow: auto;
+  max-height: calc(100vh - 120px);
```

Comment:
What's `120px`? 🤔

Reply:
Good point, it's a size of top margin and size of actions, but I will make it more readable and also I forgot condition here if there are no actions 😅